templates/motor-insurance/inputs/account.js

```diff
@@ -0,0 +1,33 @@
+import { html } from '/src/main.js';
+
+export default (inputs, outputs, cache, local, _) => html`
+    <div name="account" class="field-set">
+        <span class="field__name">Your contact info</span>
+        <p>In order to complete your policy, ${_} requires your contact details</p>
```

Comment:
The variable `_` could be more descriptive.

Reply:
that's a grey area... the only data it serves is `otp` and `serviceName` I wasn't sure it will stay this way as you move things around. No objections to make it more descriptive though just didn't want to touch things that might change 👌

Reply:
I'm not sure this even works, clarifying the creator atm